backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -83,20 +83,26 @@ async def get_speaker_service_status(current_user: User = Depends(current_superu
-
+# Memory Configuration Management Endpoints Removed - Project uses config.yml exclusively
 @router.get("/admin/memory/config/raw")
 async def get_memory_config_raw(current_user: User = Depends(current_superuser)):
-    """Get current memory configuration YAML as plain text. Admin only."""
+    """Get memory configuration YAML from config.yml. Admin only."""
     return await system_controller.get_memory_config_raw()
 
+@router.post("/admin/memory/config/validate")
+async def validate_memory_config_raw(
+    config_yaml: str = Body(..., media_type="text/plain"),
+    current_user: User = Depends(current_superuser),
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)